### ISSUE TITLE
(Bug report) image go to subnote of main after delete it 

### ISSUE THREAD

**Issue by sdfsfsfwrt** [zadam/trilium#4043]
### Trilium Version

0.60.4

### What operating system are you using?

Ubuntu

### What is your setup?

Local (no sync)

### Operating System Version

22

### Description

paste or insert  image to any note 
then remove it 
image go to note submenu

pay attention to gif

[<a href="https://uupload.ir/" target="_blank"><img src="https://s8.uupload.ir/files/peek_2023-06-19_12-13_dlzz.gif" border="0" alt="آپلود عکس" /></a>](url)

**Comment by zadam** [comment]
This is expected. In trilium even images are kind of notes, by default stored as children of the text note where they are placed. There's an "optimization" to hide such image notes from the tree if its included in the parent. 

Note that this is going to change, see: #3682

**Comment by sdfsfsfwrt** [comment]
Ok i understand your philosophy about image "everything is note"
I decide to remove this image from this note but it go to subnote why?
When i remove image mean that i don't want it being in my note but trilium move it to subnote 
I think this is bug rather than a feature

**Comment by Nriver** [comment]
It's not a bug, but rather a deliberate design choice. When you paste an image in Trilium, it creates a subnote to store the image, which is initially hidden. If you delete the image from the note, the hidden subnote becomes visible.

The image is a subnote from its very existence.

**Comment by sdfsfsfwrt** [comment]
Have way to turn this off

**Comment by zadam** [comment]
No. Without it appearing in the tree there would be no way to access the image. But anyway this "problem" will disappear in the next version - 0.61

**Comment by eliandoran** [comment]
The introduction of attachments instead of hidden images solves this issue by design.
